src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentDocumentClassifierPass.cs

```diff
@@ -81,6 +81,7 @@ protected override CodeTarget CreateTarget(RazorCodeDocument codeDocument, Razor
             @class.ClassName = computedClass;
             @class.Modifiers.Clear();
             @class.Modifiers.Add("public");
+            @class.Modifiers.Add("partial");
```

Comment:
Do we need to target this per-version? I guess not because it's not really a breaking change?

Reply:
@pranavkm - @ajaybhargavb did we build a 3.1 versioning scheme into the SDK yet?

Reply:
I think it's simpler for now if we just make this change happen to both 3.0 and 3.1 projects.

Reply:
Don't think so. 3.0 is still the latest.